pym/bob/input.py

```diff
@@ -911,8 +911,91 @@ def hasJenkinsPlugin(self):
         return False
 
     def status(self, workspacePath, dir, verbose = 0):
-        print("CVS SCM status not implemented!")
-        return 'error'
+        workDir = os.path.join(workspacePath, dir)
+        cvsDir = os.path.join(workDir, 'CVS')
+        if not os.path.exists(workDir) or not os.path.exists(cvsDir):
+            return 'error'
```

Comment:
I think so too. If the workDir isn't existing return empty. If you return error the `_cookCheckoutStep` will try to move a not existing directory to attic. If there is no CVS dir you can maybe test if the workDir is empty and only if it's not return error. For `error` and `unclean` the directory is moved to attic if build with the `--clean-checkout` argument.